www/css/cytube.css

```diff
@@ -173,24 +155,14 @@
     overflow-y: scroll;
 }
 
-#plmeta {
-    border: 1px solid #aaaaaa;
-}
+
 
 #pllength {
     float: right;
 }
 
-.queue_entry {
```

Comment:
Is there any particular reason to get rid of the default properties (e.g. `line-height`, `padding`, `font-size`)?  Seems like the only thing that should be overridden by the theme in this case is the border color.